code/modules/mob/living/carbon/carbon.dm

```diff
@@ -496,6 +496,10 @@ var/const/GALOSHES_DONT_HELP = 8
 	if( (!mask_only && head && (head.flags & HEADCOVERSMOUTH)) || (!head_only && wear_mask && (wear_mask.flags & MASKCOVERSMOUTH)) )
 		return 1
 
+/mob/living/carbon/proc/cannot_facehug(head_only = 0, mask_only = 0)
+	if( (!mask_only && head && (head.flags & CANNOTFACEHUG)) || (!head_only && wear_mask && (wear_mask.flags & CANNOTFACEHUG)) )
+		return 1
+

```

Comment:
Mmm, Probably true. But I guess anyway if someone wants to add masks for facehugger protection, they can do it themselves easily in the future.